src/main/java/quotify_app/data_access/PredictionDataAccessObjectLiveTest.java

```diff
@@ -0,0 +1,202 @@
+package quotify_app.data_access;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import quotify_app.data_access.exceptions.PredictionClientException;
+import quotify_app.entities.PredictionRequest;
+import quotify_app.entities.regionEntities.Address;
+import quotify_app.entities.regionEntities.Identifier;
+import quotify_app.entities.regionEntities.Property;
+import quotify_app.entities.regionEntities.Summary;
+
+/**
+ * Test class for the PredictionDataAccessObject.
+ */
+class PredictionDataAccessObjectLiveTest {
+    private PredictionDataAccessObject dataAccessObject;
+    private final int numPredictions = 6;
+    private final Address address = new Address(
+            "US",
+            "CA",
+            "San Francisco",
+            "Market St",
+            "123",
+            "94103"
+    );
+    private final Identifier identifier = new Identifier(
+            "123456",
+            Map.of(
+                    "ZI", "00b6f7372d30b75c08e761bf9f975fa4",
+                    "CS", "123",
+                    "ST", "CA",
+                    "CO", "SomeCounty",
+                    "N2", "Neighborhood"
+            )
+    );
+    private final Summary summary = new Summary(
+            "SFR",
+            3,
+            2,
+            "Good",
+            1,
+            1500,
+            2000
+    );
+
+    @BeforeEach
+    void setUp() {
+        // Initialize the real PredictionClient and DataAccessObject
+        final PredictionClient client = new PredictionClient();
+        dataAccessObject = new PredictionDataAccessObject(client);
+    }
+
+    @Test
+    void testGetCurrentPricePredictionPass() throws PredictionClientException {
+        // Mock property data
+
+        final Property property = new Property(identifier, address, summary);
+
+        // Call the live API for current price prediction
+        final double prediction = dataAccessObject.getCurrentPricePrediction(property);
+
+        // Assert that the prediction is a valid numerical value
+        Assertions.assertTrue(prediction > 0, "Prediction should be a positive number");
+        System.out.println("Current Price Prediction: " + prediction);
+
+    }
+
+    @Test
+    void testGetFuturePricePredictionsPass() throws PredictionClientException {
+
+        final Property property = new Property(identifier, address, summary);
```

Comment:
Initialize your Property in the @BeforeEach method so all tests can use it.
put the bottom line in ur setUp() function.
property = new Property(identifier, address, summary);

Reply:
I added it as a class variable to fix this instead, since placing it in the beforeeach method made it inaccessible to the other tests somehow lol